Lets Do This/LetsDoThis.pch

```diff
@@ -12,6 +12,8 @@
 // Include any system framework and library headers here that should be included in all compilation units.
 // You will also need to set the Prefix Header build setting of one or more of your targets to reference this file.
 
+#import <Foundation/Foundation.h>
```

Comment:
Let's add a TODO comment here about how we need to research why adding these here (or at all) doesn't seem to make a difference.